news.html

```diff
@@ -17,7 +17,9 @@ <h5 class="mb-0">
              href="https://docs.dockstore.org/en/stable/getting-started/getting-started-with-notebooks.html">
               Notebooks</a>
       </h5>
-        <span>Learn how to register notebooks in Dockstore</span>
+        <span>Learn how to register notebooks in Dockstore. For a fun example of a notebook on Dockstore, explore this <a target="_blank" rel="noopener noreferrer"
```

Comment:
Hmmm, was thinking about more of a discussion. 
Do we have a different tutorial notebook that we can point to so we can describe that before describing the more complex TRS notebook example?  
We could even consider two news items one for registration and a simple notebook example, and a second one for the TRS notebook.

Reply:
This is follow-up for https://oicr.slack.com/archives/C05EZH3RVNY/p1707412377242599